client/src/containers/SignupForm.jsx

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import { Box, Button, Image, Text } from "grommet";
+import { DfModal, DfSelectField, DfTextField } from "../components";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+
+const SignupForm = () => {
+  const [open, setOpen] = useState(false);
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    setOpen(true);
+  };
+
+  const navigate = useNavigate();
+
+  const actionButtons = [
+    {
+      label: "Cancel",
+      onClick: () => {
+        setOpen(false);
+        navigate("/error");
+      },
+    },
+    {
+      primary: true,
+      label: "Accept and Register",
+      onClick: () => {
+        setOpen(false);
+        navigate("/success");
+      },
+    },
+  ];
+
+  return (
+    <Box
+      pad="medium"
+      direction="row"
+      flex="grow"
+      align="start"
+      justify="center"
+      responsive={false}
+    >
+      <Box direction="column" align="start" width="100%">
+        <Text size="xxlarge" weight="bold" color="text-strong">
+          Sign Up
+        </Text>
+        <Text size="small" color="text-strong">
+          Sign Up for GreenLake for Data Fabric beta access
+        </Text>
+
+        <form
+          onSubmit={handleSubmit}
+          data-testid="signupForm"
+          className="DfForm full"
+          style={{
+            display: "flex",
+            flexDirection: "column",
+            width: "100%",
+          }}
+        >
+          <DfTextField
+            id="signUpName"
+            name="name"
+            label="Name"
+            required
+            // error="Name is required"
+          />
+          <DfTextField
+            id="signUpSurname"
+            name="Surname"
+            label="Surname"
+            required
+            type="Surname"
+            // error="Surname is required"
+          />
+          <DfTextField
+            id="signUpEmail"
+            name="Email"
+            label="Email"
+            required
+            type="Email"
+            error="Email is required"
```

Comment:
Looks like the validation isn't working. It always shows an error:
<img width="586" alt="image" src="https://user-images.githubusercontent.com/84544729/204342300-0254703e-33e4-48ce-bf17-548942b81586.png">